src/en/howto-privatecloud.md

```diff
@@ -1,6 +1,6 @@
-Title: Setting up private clouds with Simplestreams
+Title: Setting up image streams for private clouds with Simplestreams
```

Comment:
Maybe "image metadata" rather than "image streams", for people who are unfamiliar with simplestreams?